package.json

```diff
@@ -17,6 +17,7 @@
     "create-react-class": "^15.7.0",
     "csstype": "^3.0.11",
     "customize-cra": "^1.0.0",
+    "package": "^1.0.1",
```

Comment:
Is there a reason you added this package? If not, could you remove and re-lock.

Reply:
I don't know what that package is to be honest, I did not import it knowingly.